src/Test.UnitTests.Sarif/Readers/PropertyBagConverterTests.cs

```diff
@@ -111,6 +111,30 @@ public void PropertyBagConverter_RoundTripsGuidPropertyWithoutBraces()
             _roundTrippedObject.GetProperty<Guid>("g").Should().Be(expectedGuid);
         }
 
+        [Fact]
+        [Trait(TestTraits.Bug, "1581")]
+        public void PropertyBagConverter_RoundTripsNullValueForReferenceType()
+        {
+            _input = "{\"properties\":{\"a\":null}}";
+
+            PerformRoundTrip();
+
+            _inputObject.GetProperty<string>("a").Should().BeNull();
+            _roundTrippedObject.GetProperty<string>("a").Should().BeNull();
+        }
+
+        [Fact]
+        [Trait(TestTraits.Bug, "1581")]
+        public void PropertyBagConverter_RoundTripsNullValueForValueType()
+        {
+            _input = "{\"properties\":{\"a\":null}}";
+
+            PerformRoundTrip();
+
+            Assert.Throws<InvalidOperationException>(() => _inputObject.GetProperty<int>("a"));
+            Assert.Throws<InvalidOperationException>(() => _roundTrippedObject.GetProperty<int>("a"));
```

Comment:
>Assert [](start = 12, length = 6)

For uniformity with the rest of our test code, please use FluentAssertions for this:
```
Action action = () => _inputObject.GetProperty<int>("a");
action.ShouldThrow<InvalidOperationException>();

action = () => _roundTrippedObject.GetProperty<int>("a");
action.ShouldThrow<InvalidOperationException>();
```

Reply:
As discussed, changed to action.Should().Throw<T>()